dotnet/samples/Demos/ModelContextProtocolClientServer/MCPClient/Samples/ChatCompletionAgentWithMCPToolsSample.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+using ModelContextProtocol.Client;
+
+namespace MCPClient.Samples;
+
+/// <summary>
+/// Demonstrates how to use <see cref="ChatCompletionAgent"/> with MCP tools represented as Kernel functions.
+/// </summary>
+internal sealed class ChatCompletionAgentWithMCPToolsSample : BaseSample
```

Comment:
May remove all the `MCP` as prefixes, since everything here is MCP, or keep them as prefixes for all samples.

```suggestion
internal sealed class MCPChatCompletionAgentToolsSample : BaseSample
```

Reply:
I would rather keep them with MCP prefixes because, without the prefix, some of the samples may sound confusing: "SamplingSample", etc.